package.json

```diff
@@ -103,5 +100,8 @@
   },
   "volta": {
     "node": "20.9.0"
+  },
+  "devDependencies": {
+    "vite-svg-loader": "^5.1.0"
```

Comment:
AFAIK we can have all dependencies in `dependencies` because we are only using the bundle to create the release